Directory.Build.props

```diff
@@ -36,13 +36,15 @@
         - src/mono/msbuild/common/MonoAOTCompiler.props
         - src/tasks/AppleAppBuilder/Xcode.cs
         - src/tasks/MobileBuildTasks/Apple/AppleProject.cs
+        - src/tasks/Crossgen2Tasks/Microsoft.NET.CrossGen.targets (external)
```

Comment:
It is not clear what "(external)" means here

Reply:
Delete it?

Reply:
it decided to add that based on my explanation that these targets are used/shipped outside of the repo (i.e. can't use the properties from the root Directory.Build.props). let me remove it